projects/ngx-d3timeline/src/lib/axis.service.ts

```diff
@@ -10,13 +10,15 @@ import { TimeAxisViewModel } from './axis/time-axis-view-model';
 export class AxisService {
   private dataSubject = new BehaviorSubject<TimelineEvent[]>(null);
 
-  resourcesAxisVm$ = this.dataSubject.pipe(
-    map(data => new ResourcesAxisViewModel(data, this.viewService.timelineView))
+  axis$ = combineLatest(this.dataSubject, this.viewService.view$).pipe(
```

Comment:
FYI this has a lint error: 
combineLatest is deprecated: Pass arguments in a single array instead `combineLatest([a, b, c])` (deprecation)tslint(1)